server.js

```diff
@@ -1,5 +1,8 @@
 import express from "express";
 import cors from "cors";
+import netflixTitlesData from "./data/netflix-titles.json";
+
+console.log(netflixTitlesData);
 
 // If you're using one of our datasets, uncomment the appropriate import below
 // to get started!
```

Comment:
Since you're a pro you could remove this, part from this your code is excellent!